File: geek-university/algoritmo-e-logica-de-programacao/c/exercicio_23.c
#include <stdio.h>

int main() {
    float indice;

    printf("Informe o indice de poluicao: ");
    scanf("%f", &indice);

    if (indice >= 0.3 && indice < 0.4) {
        printf("As empresas do 1 grupo devem parar as atividades.");
    } else if (indice >= 0.4 && indice < 0.5) {
        printf("As empresas do 1 e 2 grupos devem parar as atividades.");
    } else if (indice >= 0.5) {
        printf("Todas as empresas devem parar suas atividades.");
    } else {
        printf("Niveis aceitaveis.");
    }
}

File: geek-university/algoritmo-e-logica-de-programacao/python/exercicio_23.py
indice = float(input("Informe o indice de poluicao: "))

if 0.3 <= indice < 0.4:
    print("As empresas do 1 grupo devem parar as atividades.")
elif 0.4 <= indice < 0.5:
    print("As empresas do 1 e 2 grupos devem parar as atividades.")
elif indice >= 0.5:
    print("Todas as empresas devem parar suas atividades.")
else:
    print("Níveis aceitáveis.")
